examples/verify-rollup/src/partial-handlers/rollup.ts

```diff
@@ -13,10 +13,40 @@ import {
 } from '../lib/utils';
 import { zeroAddress } from 'viem';
 import { SequencerInbox__factory } from '@arbitrum/sdk/dist/lib/abi/factories/SequencerInbox__factory';
+import { currentMainnetWasmModuleRootIndex, WASMModuleRoots } from '../lib/constants';
 
 // Constants
 const minConfirmPeriodBlocks = BigInt((24 * 60 * 60) / 12.5); // 1 day
 
+export const wasmModuleRootHandler = async (
+  orbitHandler: OrbitHandler,
+  rollupAddress: `0x${string}`,
+): Promise<string> => {
+  let warningMessage: string = '';
+
+  // get the wasmModuleRoot of the given orbit chain
+  const moduleRoot = (await orbitHandler.readContract(
```

Comment:
I think we can add this check inside the `rollup.ts/rollupHandler()` function, instead of creating a new handler for it. What do you think? For now we can run all verifications from verifyRollup, I think.

Reply:
I am wondering we can have a separate command to just verify the wasm module root, so I created a new handler here, do you think we don't need to have a separate one? (Because we can also use this to check if they upgrade to our latest arbos version in the future, this is the reason why I created `currentMainnetWasmModuleRootIndex `)

Reply:
I don't see a clear reason to have it in a separate command for now. I could be wrong though. If we want to check if a team upgraded to the latest ArbOS version we can run the whole rollup verifier to see if the rest of the parameters are properly set too. In any case, this is not very important now, since there will be changes while we integrate the script in the Orbit SDK, so not 100% sure about this, haha.

Reply:
Ok, so I will add it to `rollup.ts/rollupHandler()` first